src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -15,9 +14,10 @@
     public static final String COMMAND_WORD = "clear";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Clears all entries of tutors or students from the database. "
+            + ": Clears all entries of tutors or students from the database "
```

Comment:
Should there be a `\n` at the end of the line here so that parameters appears on the next line?